### DISCUSSION TITLE
HistoGraph - Formatierung Markierugspunkte und Legende möglich?

### DISCUSSION THREAD

**Discussion by stony2k** [Zefau/ioBroker.jarvis#1373] [Q&A] [answered]
Hallo zusammen,

verstehe ich das richtig, dass ich in den Eigenschaften des Widgets nur die X und Y Achse Formatieren kann? Ich würde nämlich gerne 2 Punkte anpassen wollen:

1. die Größe der Punkte
2. Die Legende, sodass mir immer alle Werte und nicht nur der aktuelle, wo ich mit der Maus drauf bin, angezeigt wird

![image](https://user-images.githubusercontent.com/16526471/147403203-2e6e1b01-a86e-4ddc-8eb2-e7b72a9fa3f5.png)

**Comment by mcuiobroker** [comment]
zu1 man müsste die series-Einstellungen selbst beeinflussen. 
https://echarts.apache.org/en/option.html#series-line.symbolSize

zu2 dazu müsste man die series-tooltip-Einstellungen selbst definieren können:
https://echarts.apache.org/en/option.html#series-line.tooltip.formatter

**Reply by Zefau** [reply]
1. Kann ich als Option wie xy Achse verfügbar machen
2. Kann ich im Standard so rein bringen.

Gerne per Feature Request :-)

**Reply by stony2k** [reply]
> 1. Kann ich als Option wie xy Achse verfügbar machen
> 2. Kann ich im Standard so rein bringen.
> 
> Gerne per Feature Request :-)

Gerne. Brauchst du dafür 2 Feature Requests oder nimmst du Punkt 1 so "unter der Hand" auf? :)

Beim Feature Request: Da muss ich aber Glück haben, dass genug Leute voten oder? Die Sache ist die: Ich finde, dass so ein Vergleich im Chart der Werte essentiell wichtig ist. Aber aktuell ist es teilweise schwierig, da je nach Zeitpunkt des gesetzten Datenpunktes mal einer, mal mehr Werte im Popup angezeigt werden. Klar ich kann die Werte schon ablesen, aber bei weitem nicht so bequem.

Gibt es da Möglichkeiten dass das auch ohne 20 Abstimmungen umgesetzt wird? :D Ich warte noch auf deine Antwort und erstelle dann einen oder zwei Feature Request(s).

**Reply by Zefau** [reply]
Ich versteh deinen Punkt. Ich schau mal für v3.1. Für v3.0 will ich eigentlich nur noch Bugs fixen und das endlich Release.

**Reply by stony2k** [reply]
> Ich versteh deinen Punkt. Ich schau mal für v3.1. Für v3.0 will ich eigentlich nur noch Bugs fixen und das endlich Release.

Ja absolut verständlich. Wäre für mich eine super vertretbare Option. Wir testen ja alle fleißig, damit das Release endlich erfolgen kann :)

> #1377

Danke, habe direkt mal gevotet :D

**Comment by mcuiobroker** [comment, marked as answer]
https://github.com/Zefau/ioBroker.jarvis/issues/1377
